documentation/modules/auxiliary/dos/windows/http/rb_dos_iis_2021_31166.md

```diff
@@ -0,0 +1,61 @@
+## Vulnerable Application
+
+IIS web server on Windows 10 and Windows Server version 20H2 and 2004 without KB5003173.

```

Comment:
Hmm so issue with this is the documentation only describes the vulnerable targets but less about what the module is meant to do. It would be good to update this with this information so the documentation file can live on its own without people having to refer to the module to see what it actually does.

Reply:
Updated this in a commit I'll be pushing up in a bit.